Slider/Slider.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Slider</title>
+    <link rel="stylesheet" href="Slider.css">
+</head>
+<body>
+    <div class="header">
+        <div class="home">
+            <img src="logo.png " width="50px" height="50px" alt="">
```

Comment:
Add a value inside the currently empty alt attribute related to the description of content in the image.